src/stories/ProfileBanner/ProfileCard2/index.stories.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+import ProfileCard2 from "../../../components/ProfileBanner/profile/ProfileCard2/index";
```

Comment:
Can you change this variable to ProfileCardTwo or something like that?

https://builtin.com/data-science/variable-names

Reply:
are you talking about file naming or the component name I have imported? or for both?

Reply:
Shall we change both. It would be more clear noh. Using numbers in a variable like 1,2 is not a good convention noh. Read the article I suggested.

Reply:
sure, I  understand.
i have changed the names and have pushed the code.